pkgs/create-neon/src/cache/npm.ts

```diff
@@ -1,11 +1,16 @@
 import { Cache } from "../cache.js";
 
 export class NPM implements Cache {
-  readonly org: string | null;
+  readonly org: string;
 
   readonly type: string = "npm";
 
-  constructor(org: string | null) {
-    this.org = org;
+  constructor(pkg: string, org?: string) {
+    this.org = org || NPM.inferOrg(pkg);
+  }
+
+  static inferOrg(pkg: string): string {
+    const m = pkg.match(/^@([^/]+)\/([^/]+)/);
```

Comment:
Nit, use `.*` as the second capture group. NPM doesn't allow slashes, so this _should_ be equivalent, but if it breaks, it will break in a bit more predictable way (grabbing the entire section rather than something in the middle).